.github/workflows/codeql.yml

```
@@ -0,0 +1,71 @@
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript', 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
+        # Learn more about CodeQL language support at https://aka.ms/codeql-docs/language-support
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)